charts/audit-service/templates/deployment.yaml

```diff
@@ -35,40 +35,51 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       containers:
-        - name: {{ .Chart.Name }}
-          image: "{{ .Values.global.repository }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
-          ports:
-            - name: web
-              containerPort: 8089
-              protocol: TCP
-          livenessProbe:
-            httpGet:
-              path: /actuator/health
-              port: 8089
-            initialDelaySeconds: 10
-            failureThreshold: 5
-            periodSeconds: 20
-            timeoutSeconds: 5
-          readinessProbe:
-            httpGet:
-              path: /actuator/health
-              port: 8089
-            initialDelaySeconds: 10
-            failureThreshold: 5
-            periodSeconds: 20
-            timeoutSeconds: 5
-          resources:
-          {{- toYaml .Values.resources | nindent 12 }}
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.global.repository }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        ports:
+          - name: web
+            containerPort: 8089
+            protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /actuator/health
+            port: 8089
+          initialDelaySeconds: 10
+          failureThreshold: 5
+          periodSeconds: 20
+          timeoutSeconds: 5
+        readinessProbe:
+          httpGet:
+            path: /actuator/health
+            port: 8089
+          initialDelaySeconds: 10
+          failureThreshold: 5
+          periodSeconds: 20
+          timeoutSeconds: 5
+        resources:
+          {{- toYaml .Values.resources | nindent 10 }}
+        env:
+        - name: "LOADER_PATH"
+          value: {{ .Values.global.persistence.classpathJars.mountPath }}/{{ .Chart.Name }}/{{ .Chart.Version }}
```

Comment:
Should be: 
{{ .Values.global.persistence.classpathJars.mountPath }}/{{ .Chart.Name }}/classpath/{{ .Chart.Version }}
similarly everywhere else (I'm not going to label every one).